components/vf-global-header/vf-global-header.config.yml

```diff
@@ -6,4 +6,7 @@ status: live
 context:
   component-type: block
 
-  text: Visual Framework for Life Science websites
+  global_logo:
```

Comment:
This one's probably another breaking change if you're already using `text`?

Reply:
I guess - using the `{{text}}` in the `.njk` felt like a 'hack' because you have the logo that you add text to.

So this will fix that.

Still needs work to make this container component fully customisable. 

Happy to update to a breaking change if you give this a 👍

Reply:
sounds good 👍